"docs: added comment to run script as root" (#2494)
mendersoftware/mender-docs: 01.Get-started/05.Monitor/docs.md

systemctl start countdown.service
```

> You can for example put the above code in a file called `setup-countdown.sh` and run that with sudo:

Reviewer: Please rewrite to not use passive voice :+1:

Reviewer 2: @lluiscampos What do you mean? This is not passive voice.

> You can for example put the above code in a file with the name `setup-countdown.sh` and run it with sudo:

Reviewer: @olehermanse What I meant is something like
```suggestion
In order to run the script privileged, put the above code in a file with the name `setup-countdown.sh` and run it with sudo:
```

So that we avoid the "you can do this", "you may do that".

Our docs owner @estenberg has been vocal about not using these constructions of passive voice.

Reviewer 2: > So that we avoid the "you can do this", "you may do that".

@lluiscampos Ah, I see. As far as I know, this is not passive voice. Passive voice would be like this:

```
The script below may be run as root (by you).
```

Passive voice means the sentence is turned around so the object comes first and subject comes later ("you can do this" vs "this can be done by you").

Avoiding "you can do", "you may do", is about using shorter, more direct language, in this case with imperative tense (commands - do this, put that, run that).

Nothing wrong with your suggested wording. I think I suggested the wording Victor wrote, because as far as I know, most of our users reading this will know how to run a command / script as root. If not, here is an example of one way it could be done.
